Makefile

```diff
@@ -4,18 +4,18 @@ build:
 	pipenv install --dev
 
 lint:
-	pipenv check
+	pipenv check --ignore=67895
```

Comment:
Included until dependabot fixes are done. Consider this is only tweaking a test I see no reason to block it. I will add a not to remove ignore on other PR